packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -53,14 +54,26 @@ export default class Editor {
      * @param contentDiv The DIV HTML element which will be the container element of editor
      * @param options An optional options object to customize the editor
      */
-    constructor(contentDiv: HTMLDivElement, options: EditorOptions = {}) {
+    constructor(contentDiv: HTMLDivElement, options: EditorOptions = {}, darkMode: boolean | DarkModeOptions = false) {

```

Comment:
Do you think this function signature is sensible?

Reply:
Do you need the two types? Can you just always require DarkModeOptions and initialize defaultFormat if that field is null?

Reply:
Also, do you expect there to be more Dark Mode themes in the future? So that the backgroundColor and textColor will be different than default?

Reply:
I'd like to make sure that any consumer of rooster will be able to to define their own dark mode color pallet if they want. They also should be able to override the paste handler if they choose. I think the right call is to wrap everything in the editor options, even though one of the parameters will essentially be a no-op currently (but may be later if we implement switching).